index.css

```diff
@@ -0,0 +1,186 @@
+body {
+    background-color: #f2f2f2;
+    color: #333;
+    font-weight: 400;
+    line-height: 1.3;
+    margin: 32px;
+    font-family: Arial, Arial, Helvetica, sans-serif;
+}
+
+img {
+    max-width: 100%;
+}
+
+.nav {
+    box-sizing: border;
+    padding: 15px;
+}
+
+.topnav {
+    list-style-type: none;
```

Comment:
On lines 41-46 below, you have the `u-list-reset` class, but here you also repeat its styles. Instead of repeating those styles here, you could apply two classes to your `ul` in the HTML file, like so:
```html
<ul class="u-list-reset topnav">
```